util/singleton.h

```diff
@@ -41,6 +48,55 @@ inline T& ThreadLocalSingleton() {
   thread_local static T instance;
   return instance;
 }
+#else
+template <typename T>
+class ThreadLocalSingletonInternals final {
+ private:
+  static pthread_key_t& KeyStaticStorage() {
+    static pthread_key_t key;
+    return key;
+  }
+
+  static pthread_once_t& KeyIsInitialized() {
```

Comment:
The name of this function reads as if it returns a `bool`.

I gave it a though as a reader of this code -- why not call this function some `pthread_once_t_static_storage()` and the above one `pthread_key_t_static_storage()`?

(Personally, I would be fine with either underscores of CamelCase here, it's just to avoid the "*Is*" function **ВНЕЗАПНО** returning a reference of a weird type.)